README.md

```diff
@@ -1,41 +1,92 @@
-# ORAS Documentation
 
-[![Netlify Status](https://api.netlify.com/api/v1/badges/db61db6e-a953-4b02-b5fb-7f04f018b9d8/deploy-status)](https://app.netlify.com/sites/oras-project/deploys)
+<img src="https://github.com/oras-project/oras-www/raw/main/docs/assets/images/oras.png" alt="Oras logo" height ="auto" width="200" />
+<br />
 
-![ORAS](https://github.com/oras-project/oras-www/raw/main/docs/assets/images/oras.png)
+<h1>ORAS Documentation</h1>
+ 
+An ORAS is the web site and documentation for the ORAS project.
```

Comment:
```suggestion
This repository acts as a source for the ORAS website and documentation.
```

The previous sentence suggests that ORAS is a web site, but it is not.

Reply:
I would just say "This repository _is_ the source for the oras website and documentation."